templates/kubernetes/terraform/modules/kubernetes/backend-service.tf

```diff
@@ -0,0 +1,17 @@
+data "aws_secretsmanager_secret_version" "cf_keypair" {
+  secret_id = data.aws_secretsmanager_secret.cf_keypair.id
```

Comment:
I remember you telling me users will have to manually generate the keypair 
we should probably put a comment here indicating that its manual, 

Question: when its created it goes straight into secret-manager? or it is something we can help them put in secret manager?

Reply:
I'm a little unclear on this myself; great question!
It's probably a better experience to have them provide it locally to Zero then we create the secret.  That also removes the jankiness of having to format the secret in a compatible way.

Reply:
Yeah we definitely need instructions here. Maybe for now we just have a little script that imports the keys and writes aws secrets?